README.MD

```diff
@@ -106,6 +106,7 @@ Another option would be to gift the game to people working on UniverseLAN projec
 
 | Game                                      | Status                    | Tested GOG Galaxy SDK Version | Notes |
 | :---------                                | :-------:                 | :---------:                   | :---------: |
+| ENDLESS™ Space 2                          | :white_check_mark:(2)     | 1.152.8.0                     | Tested with build 487 |
```

Comment:
Either 1.152.0 or 1.152.8 but not 1.152.8.0 , .0 is added by Windows as it has 4 groups for a version.